components/22.04/install_codedeploy_agent.yml

```diff
@@ -17,12 +17,13 @@ phases:
           commands:
             - cd /tmp
             - wget https://aws-codedeploy-us-east-1.s3.us-east-1.amazonaws.com/releases/codedeploy-agent_1.3.2-1902_all.deb
-            - mkdir codedeploy-agent_1.3.2-1902_ubuntu22
-            - dpkg-deb -R codedeploy-agent_1.3.2-1902_all.deb codedeploy-agent_1.3.2-1902_ubuntu22
-            - sed 's/Depends:.*/Depends:ruby3.0/' -i ./codedeploy-agent_1.3.2-1902_ubuntu22/DEBIAN/control
-            - dpkg-deb -b codedeploy-agent_1.3.2-1902_ubuntu22/
-            - sudo dpkg -i codedeploy-agent_1.3.2-1902_ubuntu22.deb
-
+            - VERSION=$(aws s3 ls s3://aws-codedeploy-eu-west-2/releases/ | grep deb | sort --reverse | head -n 1 | cut -c 49-58)
```

Comment:
Oh, I didn't see this, but `s3://aws-codedeploy-eu-west-2/latest/` exists and saves some effort here…